tests/labtracks/test_client.py

```diff
@@ -285,19 +285,16 @@ def close():
 
         type(mock_connect.return_value).cursor = MockCursor
 
-        actual_status_code, actual_models = self.lb_client.get_subject_info(
-            subject_id
-        )
+        actual_response = self.lb_client.get_subject_info(subject_id)
+        actual_response_json = actual_response.map_to_json_response()
         expected_subject2 = TestResponseExamples.expected_subject.copy()
         expected_subject2.sex = Sex.FEMALE
-        (
-            expected_status_code,
-            expected_models,
-        ) = Responses.multiple_items_found_response(
-            [TestResponseExamples.expected_subject, expected_subject2]
+
+        self.assertEqual(StatusCodes.DB_RESPONDED, actual_response.status_code)
+        self.assertEqual(
+            StatusCodes.MULTIPLE_RESPONSES.value,
+            actual_response_json.status_code,
```

Comment:
There should be an assertEqual between the actual_response.aind_model and the expected model response here, like the other unit tests.